Wikipedia/Code/ArticleContextMenuPresenting.swift

```diff
@@ -29,7 +29,10 @@ extension ArticleContextMenuPresenting {
             completionHandler(nullConfig)
         }
 
-        guard let linkURL = elementInfo.linkURL else {
+        // This "N/A" part is pretty hacky. But we don't want to do a preview for "view article in browser", and this is the URL that is sent
+        // for "view article in browser". Without this "N/A" check, "View article in browser"'s preview and context menu shows the one for the
+        // article "N/A", which seems more wrong. The trade off: previews for the actual article "N/A" don't work, either.
+        guard let linkURL = elementInfo.linkURL, !linkURL.absoluteString.hasSuffix("wikipedia.org/wiki/N/A") else {
```

Comment:
@mcleinman One more suggestion - it might be better to check with `linkURL.wmf_title != "N/A"` instead of hardcoding the host. Our beta cluster environment structures URLs like `https://en.wikipedia.beta.wmflabs.org/wiki/N/A` and the PCS staging environment structures it like `https://mobileapps.wmflabs.org/en.wikipedia.org/v1/page/mobile-html/N/A` (note this latter one seems to be down, which is a separate issue).

Reply:
Updated, thanks!